### PR TITLE
Feature/piechart

### PR DESCRIPTION
**Pull request by emilg1101**

### CODE REVIEW

**Comment by emilg1101** [comment]
image

File: columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
+package com.github.jaskelai.chartcustomview
+
+import android.content.Context
+import android.graphics.*
```
Comment (Jaskelai) [inline columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt]: Почисть импорты

File: columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+import android.graphics.drawable.ColorDrawable
+import android.util.AttributeSet
+import android.view.View
+import androidx.appcompat.widget.TintTypedArray
+import java.util.*
```
Comment (Jaskelai) [inline columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt]: same

File: columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        }
+    }
+
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
+
+        val size: Int
+        val width = measuredWidth
+        val height = measuredHeight
+        val widthWithoutPadding = width - paddingLeft - paddingRight
```
Comment (Jaskelai) [inline columnchartview/src/main/java/com/github/jaskelai/chartcustomview/PieChartView.kt]: support RTL?

**Comment by emilg1101** [comment]
lya kakoy

**Pull request merged**